src/modules/account/components/AccountDetails/AccountOverview/AccountOverview.css

```diff
@@ -2,3 +2,13 @@
   padding-left: 0 !important;
   padding-right: 0 !important;
 }
+
+.bannerSwiper {
+  & :global(.swiper-pagination-bullet) {
+    border: 1px solid var(--color-periwinkle-blue);
+
+    &:global(.swiper-pagination-bullet-active) {
+      background-color: white;
+    }
+  }
```

Comment:
Also add 
```
  & :global(.swiper-pagination-bullets) {
    text-align: initial;
    width: initial;
    left: 30px;
  }
  ```

For this:  
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/7d9fe365-bcf4-41b2-a733-90821741495d)

Reply:
Done